oshmem/mca/memheap/base/base.h

```diff
@@ -36,13 +36,14 @@ OSHMEM_DECLSPEC int mca_memheap_base_select(void);
 #define MEMHEAP_BASE_MIN_ORDER         3                                /* forces 64 bit alignment */
 #define MEMHEAP_BASE_PAGE_ORDER        21
 #define MEMHEAP_BASE_PRIVATE_SIZE      (1ULL << MEMHEAP_BASE_PAGE_ORDER) /* should be at least the same as a huge page size */
-#define MEMHEAP_BASE_MIN_SIZE          (1ULL << MEMHEAP_BASE_PAGE_ORDER)    /* must fit into at least one huge page */
+#define MEMHEAP_BASE_MIN_SIZE          (1ULL << MEMHEAP_BASE_PAGE_ORDER) /* must fit into at least one huge page */
```

Comment:
I realize that this is a cherry-pick from master and therefore we can't really fix this here, but can you please try to avoid spurious whitespace changes in commits that contain code/logic changes in the future?

Whitespace-only changes are fine, but please fix those in a separate commit that has no other code/logic changes.  Doing it this way just makes it easier to see code/logic fixes (which is the important stuff).

Thanks.